### PR TITLE
Update: Variable result

### PR DESCRIPTION
**Pull request by adodge-bcg**
- updated SplunkResult to allow for variable fields in addition to message and source
- updated TeamsChannelService to append the variable results from Splunk between Owner and Message
- updated RocketchatChannelService to append the variable results from Splunk between Owner and Message

- removed unused dependency from pom

### CODE REVIEW

File: src/main/java/com/nttdata/nttdatanotificationservice/rocket/models/RocketMessage.java [hunk 1]
```diff
@@ -1,17 +1,23 @@
...
     private String text;
 
+    public RocketMessage(String alias, String avatar) {
+        this.alias = alias;
+        this.avatar = avatar;
+    }
+
     public String getAlias() { return alias; }
 
     public void setAlias(String alias) {  this.alias = alias; }
```
Comment (alexjoybc) [inline src/main/java/com/nttdata/nttdatanotificationservice/rocket/models/RocketMessage.java:19]: setters can be removed, so your object becomes immutable ;-)

File: src/main/java/com/nttdata/nttdatanotificationservice/rocket/RocketChannelService.java [hunk 2]
```diff
@@ -18,12 +18,24 @@ public ChatApp getChatApp() {
 
     @Override
     public Object generatePayload(SplunkAlert splunkAlert) {
-        RocketMessage rocketMessage = new RocketMessage();
-        String text = String.format(ROCKETCHATTEMPLATE, splunkAlert.getResult().getSource(), splunkAlert.getSearch_name(), splunkAlert.getOwner(), splunkAlert.getResult().getMessage(), splunkAlert.getResults_link());
-        rocketMessage.setText(text);
-        rocketMessage.setAlias(splunkAlert.getSearch_name());
-        rocketMessage.setAvatar("https://user-images.githubusercontent.com/51387119/82707419-ddb1c600-9c30-11ea-8bfa-b3c624b23cdd.png");
+        RocketMessage rocketMessage = RocketMessage.defaultNttMessage(splunkAlert.getSearch_name());
```
Comment (alexjoybc) [inline src/main/java/com/nttdata/nttdatanotificationservice/rocket/RocketChannelService.java:21]: nice

File: src/main/java/com/nttdata/nttdatanotificationservice/rocket/models/RocketMessage.java [hunk 3]
```diff
@@ -1,17 +1,23 @@
 package com.nttdata.nttdatanotificationservice.rocket.models;
 
 public class RocketMessage {
+
+    private static final String AVATAR = "https://user-images.githubusercontent.com/51387119/82707419-ddb1c600-9c30-11ea-8bfa-b3c624b23cdd.png";
```
Comment (alexjoybc) [inline src/main/java/com/nttdata/nttdatanotificationservice/rocket/models/RocketMessage.java:5]: duplicated, you should move that property up, both teams and rocket message are using it

File: src/main/java/com/nttdata/nttdatanotificationservice/rocket/models/RocketMessage.java (hunk 1 above)
Comment (adodge-bcg) [reply src/main/java/com/nttdata/nttdatanotificationservice/rocket/models/RocketMessage.java]: Removed

File: src/main/java/com/nttdata/nttdatanotificationservice/rocket/models/RocketMessage.java (hunk 3 above)
Comment (adodge-bcg) [reply src/main/java/com/nttdata/nttdatanotificationservice/rocket/models/RocketMessage.java]: Refactored, take a look and tell me if that is what you meant

**Comment by akroon3r** [comment]
Forced code climate approval

**Pull request merged**